<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QDevCpp Updater</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../qdevcpp/resource.qrc">
    <normaloff>:/qdevcpp.ico</normaloff>:/qdevcpp.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QFormLayout" name="formLayout">
        <item row="1" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>稳定版本</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>开发版本</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="stableVer">
          <property name="text">
           <string>v0.0</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="developVer">
          <property name="text">
           <string>v0.0</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>当前版本</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="currentVer">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="updateInfo">
          <property name="text">
           <string>更新信息</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="updateStable">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>更新至稳定版</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="updateDevelop">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>更新至开发版</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="downloadUrl">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="downloadProgress">
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../qdevcpp/resource.qrc"/>
 </resources>
 <connections/>
</ui>
